packages/smooth_app/lib/helpers/strings_helper.dart

```diff
@@ -138,3 +139,11 @@ class FormattedText extends StatelessWidget {
     );
   }
 }
+
+extension CountryExtension on Country {
```

Comment:
We already have something identical called `String? EmojiHelper().getCountryEmoji(final OpenFoodFactsCountry? country)`.